### PR TITLE
CF-USB pipeline

### PR DESCRIPTION
**Pull request by mook-as**
This is the start of the CF-USB pipeline.

Unfortunately, since the CF-USB build system makes quite a few assumptions, this pokes into it a bit.

### CODE REVIEW

File: cf-usb/config-production.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+src-repo: https://github.com/SUSE/cf-usb-sidecar.git
+src-branch: develop
+
+ci-repo: https://github.com/SUSE/cf-ci.git
+ci-branch: master
+
+docker-server: ""
+docker-tag: latest
+
+s3-bucket: cf-usb-release-blobstore
```
Comment (mook-as) [inline cf-usb/config-production.yaml]: I'm not sure this is the correct bucket to put things into.
Jan thinks we should stick it in our general catch-all bucket until we have something more organized.

File: cf-usb/cf-usb.yaml [hunk 2]
```diff
@@ -30,27 +30,31 @@ resources:
...
     tag: ((docker-tag))
+    tag_as_latest: true
     username: ((docker-username))
     password: ((docker-password))
 - name: docker.mysql-setup
   type: docker-image
   source:
     repository: ((docker-server))((docker-org))/cf-usb-sidecar-dev-mysql-setup
     tag: ((docker-tag))
+    tag_as_latest: true
```
Comment (mook-as) [inline cf-usb/cf-usb.yaml]: This should go on the output params, not resource definition, I think

Comment (andreas-kupries) [reply cf-usb/cf-usb.yaml]: Done with commit ef4a90b749f65afd3166761227f8b96c066f4122. In testing.

Comment (andreas-kupries) [reply cf-usb/cf-usb.yaml]: Works.

**Pull request merged**